src/views/Simulation/constants/settings.js

```diff
@@ -15,6 +15,14 @@ export const SIMULATION_MODES = {
   MAP: 'map',
 };
 
+export const MAP_VIEW_FILTERS = [
+  { value: 'all', label: 'All' },
+  { value: 'production_resources', label: 'Production ressources' },
```

Comment:
I think this might be a typo in the mockups: the label "Production ressources" should be "Production resources" (with one 's')